### PR TITLE
Fix/average block time - block sync issue

### PR DESCRIPTION
**Pull request by faisalnaveedRNS**
## Description
Block syncing issue resoled
Fixes #

- block sync issue 
- average block time
- settings.json remove 
- git ignore was updated
-
## Checklist
- [x] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Run `meteor npm run lint`
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Review by MikeSofaer — APPROVED** [review]

**Review by BlackPanthar — APPROVED** [review]

File: imports/api/blocks/server/methods.js [hunk 1]
```diff
@@ -463,10 +465,11 @@ Meteor.methods({
                     if (lastSyncedTime){
                         let dateLatest = new Date(blockData.time);
                         let dateLast = new Date(lastSyncedTime);
-                        let genesisTime = new Date(genesisTime);
+
+                        // calculating time to generate average block time 
+                        let genesisTimeStamp = new Date(genesisTime);
                         timeDiff = Math.abs(dateLatest.getTime() - dateLast.getTime());
```
Comment (mustafapylons) [inline imports/api/blocks/server/methods.js:471]: It doesn't look like you are using timeDiff in the calculations, are the calculations correct?

**Review by mustafapylons — COMMENTED** [review]
Critical issue to clarify about block calculations, once resolved, it can be approved.

**Review by mustafapylons — APPROVED** [review]
Approved per our discussion.

File: imports/api/blocks/server/methods.js (hunk 1 above)
Comment (faisalnaveedRNS) [reply imports/api/blocks/server/methods.js:471]: all the calculations are done in flat and on fronted these are displayed in 2 decimal

**Pull request merged**